### PR TITLE
Some code cleanup

### PR DESCRIPTION
**Pull request by NoUserAgent**
The commits can be reviewed individually since they are independent.
Most of the changes are trivial - the non-trivial ones are annotated.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: engine/src/main/java/org/terasology/audio/openAL/BaseSoundSource.java [hunk 1]
```diff
@@ -272,11 +272,4 @@ public void dispose() {
             AL10.alDeleteSources(sourceId);
         }
     }
-
-    @Override
-    public int hashCode() {
```
Comment (NoUserAgent) [inline engine/src/main/java/org/terasology/audio/openAL/BaseSoundSource.java]: The equals method is not implemented. I assume it makes most sense to go with defaults.

**Comment by msteiger** [comment]
ok to test

**Comment by GooeyHub** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.terasology.org/job/TerasologyPRs/315/
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Thanks! :-)

**Pull request merged**